Zadanie 12.4 Ożywiamy kanban!/js/Card.js

```diff
@@ -0,0 +1,37 @@
+function Card(id, name) {
+	var self  = this;
+	
+	this.id = id;
+	this.name = name || 'No name given';
+	this.$element = createCard();
+	
+	function createCard() {
+		 var $card = $('<li>').addClass('card');
+		 var $cardDescription = $('<p>').addClass('card-description').text(self.name);
+		 var $cardDeleteButton = $('<button>').addClass('card-delete btn');
+		 var $cardDeleteSymbol = $('<span  class="glyphicon glyphicon-remove" aria-hidden="true" aria-hidden="true"></span>');
```

Comment:
tutaj uzywasz klas z bootstrapa, ale bootstrapa nie ma dolaczonego, wiec nie pojawia sie zadna ikona do usuniecia karty. Mozesz to uproscic na wzor usuwania kolumn